Mixpanel/Mixpanel.m

```diff
@@ -96,8 +96,6 @@ - (instancetype)initWithToken:(NSString *)apiToken launchOptions:(NSDictionary *
         self.telephonyInfo = [[CTTelephonyNetworkInfo alloc] init];
 #endif
 #endif
-        MPSetLoggingEnabled(YES);
```

Comment:
this was a duplicate method of `Mixpanel setEnableLogging:`. `enableLogging` promised to be off by default but this turned it on by default. they can still set it through the other method (not sure this one is even exposed)